.eslintrc.js

```diff
@@ -3,4 +3,9 @@ const { config } = require('@dhis2/cli-style')
 module.exports = {
     parser: 'babel-eslint',
     extends: ['eslint:recommended', 'plugin:react/recommended', config.eslint],
+    env: { es6: true },
+    globals: {
+        Cypress: 'readonly',
```

Comment:
These should only apply to the `cypress` folder, if possible.  Can we put a subsidiary `.eslintrc` in `cypress/`?  That might not be supported by cli-style...  @varl?

Reply:
Oh yeah! That'd be awesome if supported

Reply:
@Mohammer5 Any success with this?

Reply:
Yes! It's working!